lib/fputs.js

```diff
@@ -0,0 +1,185 @@
+/**
```

Comment:
General comment:  There should be validation in case callbacks are not passed or if a non-funciton is passed.